README.md

```
@@ -66,9 +66,39 @@ These branches are useful for sharing to ship to dev sandboxes, and should not b
 
 All development work should branch off of and PR into the `develop` branch.
 
+### Setting Up Your Development Environment
+
+#### Backend Setup
+
+To develop against a remote AWS dev stack:
+
+1. **Deploy your backend stack** (replace `stack` with your stack name, e.g., `john`):
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)